libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/model/RouteSetValue.kt

```diff
@@ -58,3 +58,8 @@ fun interface RouteLineExpressionProvider {
      */
     fun generateExpression(): Expression
 }
+
+/**
+ * Represents a function that returns an [Expression]
+ */
+fun interface RouteLineTrimExpressionProvider : RouteLineExpressionProvider
```

Comment:
We need better docs on why this is here and needed.

Reply:
done